pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -229,6 +237,9 @@ def destroy(self, request, repository_pk, number):
         )
         return OperationPostponedResponse(async_result, request)
 
+    @swagger_auto_schema(operation_description="Trigger an asynchronous task to create "
```

Comment:
It would be good if the other descriptions would have the same level of detail w/r/t what they are updating or deleting (a Repository Version).

Otherwise, LGTM